swift/ql/lib/codeql/swift/dataflow/internal/DataFlowImplSpecific.qll

```diff
@@ -1,9 +1,14 @@
 /**
  * Provides Swift-specific definitions for use in the data flow library.
  */
+
+import swift as Swift
```

Comment:
Could this be private?

Reply:
Yep, just realized it 😅 Fixed in 978ed03, thanks for pointing it out.